src/curated-corpus/helpers/helperFunctions.ts

```diff
@@ -174,3 +175,42 @@ export const getDisplayTopic = (
 
   return displayTopic ? displayTopic : 'N/A';
 };
+
+export const getLocalDateTimeForGuid = (guidCode: string) => {
+  const guidToUtcOffset = [
+    {
```

Comment:
might be better to put this mapping into some kind of config file, to make it obvious

Reply:
yep I meant to move it to the `definitions` file. Good catch!